### PR TITLE
b2 Doc: Fix PutObject API known limitations text

### PR DESCRIPTION
**Pull request by kannappanr**
## Description

Sending a PR to include the suggestion in #7680 

## Motivation and Context
Suggestion in #7680

## Regression
No

## How Has This Been Tested?
Documentation, so it has not been tested

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/gateway/b2.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ mc ls myb2
 ### Known limitations
 Gateway inherits the following B2 limitations:
 
-- PutObject() does not return an md5sum of the uploaded file as an etag response.  Apps that check vailidity will fail.
+- PutObject S3 API does not return md5sum of the uploaded file as it's etag response. Applications that check validity of the uploaded object by comparing these two values will fail.
```
Comment (eco-minio) [inline docs/gateway/b2.md]:
```suggestion
- PutObject S3 API does not return the md5sum of an uploaded file as it's etag response. Applications that check the validity of the uploaded object by comparing these two values will fail.
```

**Review by eco-minio — CHANGES_REQUESTED** [review]

File: docs/gateway/b2.md (hunk 1 above)
Comment (harshavardhana) [reply docs/gateway/b2.md]: We should also mention that this is regardless of `Content-Md5` being set by the client.

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#807747bf-bc38-44b4-80e9-8a51d1ffc240) |
<a name="807747bf-bc38-44b4-80e9-8a51d1ffc240"></a>
### 7704-b9ae580/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:31937
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp e4183590492b:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 1 seconds
(2/14) Running aws-sdk-java tests ... done in 2 seconds
(3/14) Running aws-sdk-php tests ... done in 49 seconds
(4/14) Running aws-sdk-ruby tests ... done in 15 seconds
(5/14) Running awscli tests ... done in 2 minutes and 6 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... FAILED in 2 minutes and 21 seconds
{
  "name": "mc",
  "duration": "118832",
  "function": "test_mirror_list_objects_storage_class",
  "status": "FAIL",
  "error": "/mint/run/core/mc/mc --config-dir /tmp/.mc-4608 --quiet --no-color mirror --storage-class REDUCED_REDUNDANCY /mint/data myminio/mc-test-bucket-28282  >>> \n`/mint/data/datafile-0-b` -> `myminio/mc-test-bucket-28282/datafile-0-b`\n`/mint/data/datafile-1-kB` -> `myminio/mc-test-bucket-28282/datafile-1-kB`\n`/mint/data/datafile-1-b` -> `myminio/mc-test-bucket-28282/datafile-1-b`\n`/mint/data/datafile-10-kB` -> `myminio/mc-test-bucket-28282/datafile-10-kB`\n`/mint/data/datafile-11-MB` -> `myminio/mc-test-bucket-28282/datafile-11-MB`\n`/mint/data/datafile-1-MB` -> `myminio/mc-test-bucket-28282/datafile-1-MB`\n`/mint/data/datafile-6-MB` -> `myminio/mc-test-bucket-28282/datafile-6-MB`\n`/mint/data/datafile-1.03-MB` -> `myminio/mc-test-bucket-28282/datafile-1.03-MB`\n`/mint/data/datafile-65-MB` -> `myminio/mc-test-bucket-28282/datafile-65-MB`\n`/mint/data/datafile-100-kB` -> `myminio/mc-test-bucket-28282/datafile-100-kB`\n`/mint/data/datafile-10-MB` -> `myminio/mc-test-bucket-28282/datafile-10-MB`\n`/mint/data/datafile-33-kB` -> `myminio/mc-test-bucket-28282/datafile-33-kB`\n`/mint/data/datafile-5-MB` -> `myminio/mc-test-bucket-28282/datafile-5-MB`\n`/mint/data/datafile-5243880-b` -> `myminio/mc-test-bucket-28282/datafile-5243880-b`\nmc: <ERROR> Failed to copy `/mint/data/datafile-65-MB`. Multiple disks failures, unable to write data.\nmc: <ERROR> Failed to copy `/mint/data/datafile-11-MB`. Multiple disks failures, unable to write data.\nmc: <ERROR> Failed to copy `/mint/data/datafile-10-MB`. Multiple disks failures, unable to write data.\nTotal: 104.17 MiB, Transferred: 371.20 MiB, Speed: 3.13 MiB/s"
}

Executed 6 out of 14 tests successfully.
```

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#1fb832cd-f2ce-44ca-86f9-ecf23dca0d47) |
<a name="1fb832cd-f2ce-44ca-86f9-ecf23dca0d47"></a>
### 7704-1ba87e1/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.59:30309
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 8029ba5603c7:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 0 seconds
(2/14) Running aws-sdk-java tests ... done in 2 seconds
(3/14) Running aws-sdk-php tests ... done in 48 seconds
(4/14) Running aws-sdk-ruby tests ... done in 13 seconds
(5/14) Running awscli tests ... done in 2 minutes and 4 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... FAILED in 2 minutes and 9 seconds
{
  "name": "mc",
  "duration": "105017",
  "function": "test_mirror_list_objects_storage_class",
  "status": "FAIL",
  "error": "/mint/run/core/mc/mc --config-dir /tmp/.mc-16271 --quiet --no-color mirror --storage-class REDUCED_REDUNDANCY /mint/data myminio/mc-test-bucket-1187  >>> \n`/mint/data/datafile-1-MB` -> `myminio/mc-test-bucket-1187/datafile-1-MB`\n`/mint/data/datafile-0-b` -> `myminio/mc-test-bucket-1187/datafile-0-b`\n`/mint/data/datafile-1-b` -> `myminio/mc-test-bucket-1187/datafile-1-b`\n`/mint/data/datafile-1.03-MB` -> `myminio/mc-test-bucket-1187/datafile-1.03-MB`\n`/mint/data/datafile-1-kB` -> `myminio/mc-test-bucket-1187/datafile-1-kB`\n`/mint/data/datafile-10-kB` -> `myminio/mc-test-bucket-1187/datafile-10-kB`\n`/mint/data/datafile-10-MB` -> `myminio/mc-test-bucket-1187/datafile-10-MB`\n`/mint/data/datafile-11-MB` -> `myminio/mc-test-bucket-1187/datafile-11-MB`\n`/mint/data/datafile-33-kB` -> `myminio/mc-test-bucket-1187/datafile-33-kB`\n`/mint/data/datafile-100-kB` -> `myminio/mc-test-bucket-1187/datafile-100-kB`\n`/mint/data/datafile-5243880-b` -> `myminio/mc-test-bucket-1187/datafile-5243880-b`\n`/mint/data/datafile-6-MB` -> `myminio/mc-test-bucket-1187/datafile-6-MB`\n`/mint/data/datafile-65-MB` -> `myminio/mc-test-bucket-1187/datafile-65-MB`\n`/mint/data/datafile-5-MB` -> `myminio/mc-test-bucket-1187/datafile-5-MB`\nmc: <ERROR> Failed to copy `/mint/data/datafile-10-MB`. Multiple disks failures, unable to write data.\nmc: <ERROR> Failed to copy `/mint/data/datafile-11-MB`. Multiple disks failures, unable to write data.\nmc: <ERROR> Failed to copy `/mint/data/datafile-65-MB`. Multiple disks failures, unable to write data.\nTotal: 104.17 MiB, Transferred: 355.91 MiB, Speed: 3.39 MiB/s"
}

Executed 6 out of 14 tests successfully.
```

**Review by harshavardhana — APPROVED** [review]

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#48e9a0a3-52b4-4097-b751-ade3db4c1292) |
| mint-dist-xl.sh | :x: [more...](#a3eb0bf2-ee77-4fa4-bdb8-3c6d1011cacd) |
<a name="48e9a0a3-52b4-4097-b751-ade3db4c1292"></a>
### 7704-27e1c44/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:31569
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp d78e36020726:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 2 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 49 seconds
(4/14) Running aws-sdk-ruby tests ... done in 12 seconds
(5/14) Running awscli tests ... done in 2 minutes and 22 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... FAILED in 1 minutes and 11 seconds
{
  "name": "mc",
  "duration": "69055",
  "function": "test_put_object_with_storage_class",
  "status": "FAIL",
  "error": "/mint/run/core/mc/mc --config-dir /tmp/.mc-20613 --quiet --no-color cp --storage-class REDUCED_REDUNDANCY /mint/data/datafile-1-MB myminio/mc-test-bucket-975/mc-test-object-24062  >>> \n`/mint/data/datafile-1-MB` -> `myminio/mc-test-bucket-975/mc-test-object-24062`\nmc: <ERROR> Failed to copy `/mint/data/datafile-1-MB`. Please reduce your request\nmc: <ERROR> Session safely terminated. To resume session `mc session resume ieXiAYgb`"
}

Executed 6 out of 14 tests successfully.
```
<a name="a3eb0bf2-ee77-4fa4-bdb8-3c6d1011cacd"></a>
### 7704-27e1c44/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.54:30935
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 54d21397b3ac:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 0 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 42 seconds
(4/14) Running aws-sdk-ruby tests ... done in 3 seconds
(5/14) Running awscli tests ... done in 1 minutes and 11 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... FAILED in 8 seconds
{
  "name": "mc",
  "duration": "4614",
  "function": "test_get_object_multipart",
  "status": "FAIL",
  "error": "/mint/run/core/mc/mc --config-dir /tmp/.mc-1337 --quiet --no-color rm mc-test-object-6506.downloaded myminio/mc-test-bucket-21107/mc-test-object-6506  >>> \nRemoving `mc-test-object-6506.downloaded`.\nmc: <ERROR> Failed to remove `myminio/mc-test-bucket-21107/mc-test-object-6506`. Target object is not found "
}

Executed 6 out of 14 tests successfully.
```

**Pull request merged**